internal/webhook/sidecar/webhookhandler.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package webhookhandler contains the webhook that injects sidecars into pods.
-package webhookhandler
+// Package sidecar contains the webhook that injects sidecars into pods.
+package sidecar
```

Comment:
I don't think we should call this package `sidecar` 

The package exposes a mutator API that allows mutating pod objects. It is used for instrumentation and sidecar